include/extractor/extractor_callbacks.hpp

```diff
@@ -84,6 +88,9 @@ class ExtractorCallbacks
 
     // destroys the internal laneDescriptionMap
     guidance::LaneDescriptionMap &&moveOutLaneDescriptionMap();
+
+    // destroys classes_map
+    ClassesMap &&moveOutClassMap();
```

Comment:
After thinking about this they should not be owned by `ExtractorCallbacks` in the first place. Going to change that.